scala/integration/intellij-bsp/resources/META-INF/scala-intellij-bsp-integration.xml

```diff
@@ -1,9 +1,13 @@
 <idea-plugin>
-    <extensions defaultExtensionNs="com.intellij">
+    <extensions defaultExtensionNs="org.jetbrains.bsp">
         <bspProjectOpenProcessorExtension
                 implementation="org.jetbrains.plugins.scala.bsp.flow.open.ScalaBspProjectOpenProcessor"/>
         <bspProjectAwareExtension implementation="org.jetbrains.plugins.scala.bsp.config.ScalaBspProjectAwareExtension"/>
-        <bspBuildTargetClassifierExtension implementation="org.jetbrains.plugins.scala.bsp.extension.ScalaBuildTargetClassifier" />
+        <buildTargetClassifierExtension implementation="org.jetbrains.plugins.scala.bsp.extension.ScalaBuildTargetClassifier" />
         <buildToolAssetsExtension implementation="org.jetbrains.plugins.scala.bsp.assets.ScalaAssetsExtension" />
     </extensions>
+
+    <extensions defaultExtensionNs="org.jetbrains.bsp">
```

Comment:
Why not append it to the previous extensions section, since the two share the same namespace? Unless we'd rather separate the scala-specific extension point implementations from the regular bsp ones